src/core/meshcop/meshcop_tlvs.hpp

```diff
@@ -1087,6 +1087,16 @@ class StateTlv : public UintTlvInfo<Tlv::kState, uint8_t>
         kPending = 0,    ///< Pending
         kAccept  = 1,    ///< Accept
     };
+
+    /**
+     * Converts a `StateTlv::State` to a string.
+     *
+     * @param[in] aState  An item state.
+     *
+     * @returns A string representation of @p aState.
+     *
+     */
+    static const char *StateToString(StateTlv::State aState); // todo probably I don't need to add StateTlv I guess, check it
```

Comment:
Yes. We should not need `StateTlv::` here since we are already inside the class `StateTlv` (so we are inside its namespace). 
```suggestion
    static const char *StateToString(State aState); 
```